src/css/styles.css

```diff
@@ -285,7 +283,7 @@
 }
 
 .villain-toolbar .tooltip .tooltiptext {
-  width: 3rem;
+  min-width: 1rem;
```

Comment:
Nice, will this affect the position ?

Reply:
This won’t affect the position of the tooltip or the text within it. The text will remain cantered, as will the tooltip but there will be 1 rem less space on either side. This will make the styling consistent with other tooltip implementations.

Reply:
Edit: left-margin update has been pushed to centre tooltip